File: two_sum/two_sum.js
var two_sum = function (nums, target) {
  document.getElementById("array").innerHTML = "[" + nums + "]";
  for (let i = 0; i < nums.length; i++) {
    for (let j = i + 1; j < nums.length; j++) {
      sum = nums[i] + nums[j];
      if (sum == target) {
        document.getElementById("answer").innerHTML = "[" + [i, j] + "]";
      }
    }
  }
};

File: two_sum/two_sum.rb
def two_sum(nums, target)
    (0..nums.length-1).each do |i|
        (i+1..nums.length-1).each do |j|
            return [i,j] if nums[i] + nums[j] == target
        end
    end
end

puts two_sum([2,7,11,15], 26)
